webroot/src/models/License/License.model.spec.ts

```diff
@@ -7,95 +7,144 @@
+            status: LicenseStatus.ACTIVE,
         };
         const license = LicenseSerializer.fromServer(data);
 
         // Test field values
         expect(license).to.be.an.instanceof(License);
         expect(license.id).to.equal(data.id);
+        expect(license.compact).to.be.an.instanceof(Compact);
+        expect(license.isPrivilege).to.equal(true);
         expect(license.issueState).to.be.an.instanceof(State);
-        expect(license.issueDate).to.equal(data.issueDate);
-        expect(license.renewalDate).to.equal(data.renewalDate);
-        expect(license.expireDate).to.equal(data.expireDate);
-        expect(license.type).to.equal(data.type);
+        expect(license.isHomeState).to.equal(false);
+        expect(license.issueState.abbrev).to.equal(data.jurisdiction);
+        expect(license.issueDate).to.equal(data.dateOfIssuance);
+        expect(license.renewalDate).to.equal(data.dateOfRenewal);
+        expect(license.expireDate).to.equal(data.dateOfExpiration);
+        expect(license.occupation).to.equal(data.licenseType);
+        expect(license.statusState).to.equal(data.status);
+        expect(license.statusCompact).to.equal(data.status);
 
         // Test methods
         expect(license.issueDateDisplay()).to.equal(
-            moment(data.issueDate, serverDateFormat).format(displayDateFormat)
+            moment(data.dateOfIssuance, serverDateFormat).format(displayDateFormat)
         );
         expect(license.renewalDateDisplay()).to.equal(
-            moment(data.renewalDate, serverDateFormat).format(displayDateFormat)
+            moment(data.dateOfRenewal, serverDateFormat).format(displayDateFormat)
         );
         expect(license.expireDateDisplay()).to.equal(
-            moment(data.expireDate, serverDateFormat).format(displayDateFormat)
+            moment(data.dateOfExpiration, serverDateFormat).format(displayDateFormat)
         );
         expect(license.isExpired()).to.equal(true);
+        expect(license.occupationName()).to.equal('Audiologist');
     });
     it('should serialize a License for transmission to server', () => {
         const license = LicenseSerializer.fromServer({
             id: 'test-id',
-            issueState: new State(),
-            issueDate: moment().format(serverDateFormat),
-            renewalDate: moment().format(serverDateFormat),
-            expireDate: moment().subtract(1, 'day').format(serverDateFormat),
-            type: LicenseType.AUDIOLOGIST,
+            compact: CompactType.ASLP,
+            type: 'privilege',
+            jurisdiction: 'al',
+            dateOfIssuance: moment().format(serverDateFormat),
+            dateOfRenewal: moment().format(serverDateFormat),
+            dateOfExpiration: moment().subtract(1, 'day').format(serverDateFormat),
+            licenseType: LicenseOccupation.AUDIOLOGIST,
+            status: LicenseStatus.ACTIVE,
         });
         const transmit = LicenseSerializer.toServer(license);
```

Comment:
Is it important that this transmit object doesn't contain any of the license data fields and just contains an id? I guess more generally, why would every other field be ignored when sending data to the server?

Reply:
This was all mostly placeholder for when the backend was further along. I'll just remove the `toServer` app code & tests for this model for now.

Reply:
Sounds good. I think that makes sense.